opnode/flags/flags.go

```diff
@@ -31,6 +31,30 @@ var (
 		Required: true,
 		EnvVar:   prefixEnvVar("ROLLUP_CONFIG"),
 	}
+	L2NodeAddr = cli.StringFlag{
+		Name:     "eth.l2",
```

Comment:
Slight nit - there should be a general 1:1 mapping between env vars and flag names

Reply:
👍 . changed the flag name here.